### ISSUE TITLE
Adding options to the dropdown menu.

### ISSUE THREAD

**Issue by Sanyam2000-dot** [ADRE9/bunk-manager-mern#104]
**Deliverables:**

- [ ] Adding the option to change the pictures for subjects and labs in the dropdown menu.

**Screenshots**
![BunkManager-Subjects](https://user-images.githubusercontent.com/53525430/113472047-ff4d2200-947d-11eb-8ebc-88b002d185f5.png)

**Comment by Artistic18** [comment]
I am currently working on this issue. 
I had a question, from where are we getting the background image data for the subject card.
@Sanyam2000-dot @ADRE9

**Comment by ADRE9** [comment]
@Artistic18  for  now the images are preset randomly from the folder svg in the server . The images are set and saved for each card and fetch along with the subject data. But we are planning to host the images from the Azure Blob storage and we are thinking of attaching an option to choose from a set of images from a gallery that will be hosted in the Azure Blob storage. Till then you need to wait. I am assigning this issue to you though.

**Comment by Artistic18** [comment]
@ADRE9 ok got it and I have already implemented the feature to upload custom image from local storage, it would be great if we could combine both local and cloud functionalities together.

**Comment by ADRE9** [comment]
Yes we are thinking to do that and if time permits we can fetch images from unsplash API too.
